src/bionics_ui.cpp

```diff
@@ -50,14 +50,14 @@ char get_free_invlet( player &p )
     return ' ';
 }
 
-void show_bionics_titlebar( WINDOW *window, player *p, bionic_menu_mode mode )
+void draw_bionics_titlebar( WINDOW *window, player *p, bionic_menu_mode mode )
 {
     werase( window );
 
-    std::string pwr = string_format( _( "Power: %i/%i" ), int( p->power_level ),
-                                     int( p->max_power_level ) );
-    int desc_length = getmaxx( window ) - utf8_width( pwr ) - 1;
-    mvwprintz( window, 0, desc_length, c_white, "%s", pwr.c_str() );
+    const std::string pwr = string_format( _( "Power: %i/%i" ), int( p->power_level ),
+                                           int( p->max_power_level ) );
+    const int pwr_str_pos = getmaxx( window ) - utf8_width( pwr ) - 1;
```

Comment:
@codemime totally agree about ```right_print()```.

Concerning ```mvwprintz```: this function has very unpleasant side effect described by Wuzzy2 in #4657 - if text doesn't fit on the current line then function continues printing it on the next line (starting from x-coordinate 0, which is a place for border typically); you can see example of that in #9808.

Due to that issue, I'd prefer to replace all instances of ```mvwprintz()``` by ```trim_and_print()``` and ```fold_and_print()```. BTW, these functions can be overloaded with some reasonable defaults for ```width``` parameter - I guess it should be ```window width - posx - 1``` but this is a subject for discussion for another PR  issue.